README.md

```diff
@@ -20,6 +21,13 @@
 * [Musicbrainz DiscIds Detector](#mb_discids_detector)
 * [Musicbrainz UI enhancements](#mb_ui_enhancements)
 
+## <a name="facebook_events_importer"></a> Import Facebook Events to MusicBrainz
+
+Add a button to import events from Facebook to MusicBrainz
+
+[![Source](https://github.com/jerone/UserScripts/blob/master/_resources/Source-button.png)](https://github.com/murdos/musicbrainz-userscripts/blob/master/facebook_events_importer.js)
+[![Install](https://raw.github.com/jerone/UserScripts/master/_resources/Install-button.png)](https://raw.github.com/murdos/musicbrainz-userscripts/master/facebook_events_importer.js)
```

Comment:
Please do not change README.md in PR, it will be (re-)generated after the merge.